auxiliary/RewardsDistributor/src/RewardsDistributor.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: UNLICENSED
+// solhint-disable meta-transactions/no-msg-sender
+pragma solidity ^0.8.13;
```

Comment:
I know you're just moving files over but why such an old version of Solidity?

Reply:
Fair enough I'll alight it with the rest. Thanks

Reply:
Aligned with the core contracts `>=0.8.11 <0.9.0;`